packages/react-core/src/Authenticator/hooks/types.ts

```diff
@@ -122,7 +122,6 @@ export type SetupTOTPBaseProps<FieldType = {}> = {
 
 export type SignInBaseProps<FieldType = {}> = {
   hideSignUp?: boolean;
-  socialProviders?: AuthenticatorMachineContext['socialProviders'];
```

Comment:
I assume removing this from the 'react-core' won't negatively affect the react web package? I have no idea how this works.

Reply:
Valid question. React web doesn't use this type yet, but the types are flexible enough that we can declare these types directly on the subcomponents in the React package when we migrate